engine-rest/engine-rest-openapi/src/main/templates/paths/task/{id}/rendered-form/get.ftl

```diff
@@ -0,0 +1,57 @@
+{
+
+  <@lib.endpointInfo
+      id = "getRenderedForm"
+      tag = "Task"
+      desc = "Retrieves the rendered form for a task. This method can be used to get the HTML
+              rendering of a
+              [Generated Task Form](${docsUrl}/user-guide/task-forms/#generated-task-forms)." />
+
+  "parameters" : [
+
+    <@lib.parameter
+        name = "id"
+        location = "path"
+        type = "string"
+        required = true
+        last = true
+        desc = "The id of the task to get the rendered form for."/>
+
+  ],
+
+  "responses" : {
+
+    <@lib.response
+        code = "200"
+        mediaType = "application/xhtml+xml"
```

Comment:
the ExceptionDto is still referenced in response by default, we should fix the macro